backend/chalice/api_server/app.py

```diff
@@ -42,7 +42,7 @@ def get_chalice_app(flask_app):
     if deployment not in ["prod"]:
         allowed_origins.extend(
             [
-                r"https://.*\.dev\.single-cell\.czi\.technology",  # remove once contract work is completed.
+                "https://canary-cellxgene.dev.single-cell.czi.technology",  # remove once contract work is completed.
```

Comment:
lgtm - do we need to include cellxgene dev as well?

Reply:
not based on the problem canary is having at the moment